xfce-base/xfce4-settings/files/xfce4-settings-4.19.1-gmodule-libs.patch

```diff
@@ -0,0 +1,35 @@
+https://bugs.gentoo.org/913864
+https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/494
```

Comment:
Put that above the filename in ebuild, so people don't have to open a second file to check.

Reply:
Done.